cypress/integration/UserCanSeeRequestList.feature.js

```diff
@@ -13,8 +13,8 @@ describe("When products are visible", () => {
     });
     cy.route({
       method: "GET",
-      url: "**/products",
-      response: "fixture:products.json",
+      url: "**/tasks",
```

Comment:
Remember to push the test that's gonna be used first before updating it.

Reply:
Done